src/TorchSharp/TorchAudio/Datasets/Utils.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using ICSharpCode.SharpZipLib.GZip;
+using ICSharpCode.SharpZipLib.Tar;
+
+namespace TorchSharp
+{
+    public static partial class torchaudio
+    {
+        public static partial class datasets
+        {
+            public static partial class utils
+            {
+                internal static void extract_archive(string from_path, string to_path)
```

Comment:
Maybe this should be a public method? And add doc comments.

Reply:
Added doc comment. `torchtext.utils.extract_archive()` is a documented API of PyTorch https://pytorch.org/text/stable/_modules/torchtext/utils.html#extract_archive
but `torchaudio.datasets.utils.extract_archive()` is not documented.